source/Trashbot.js

```diff
@@ -17,4 +17,9 @@ export default class Junkership extends Pixi.Sprite {
             this.position.x = Reference.GAME_WIDTH
         }
     }
+    onCollision(){
+        game.removeChild(this)
+        this.destroy()
+        game.score++
```

Comment:
Hmmm.. Considering how we'll eventually want each player to have their own individual score, we'll need to pass a reference of the ``Junkership`` to each ``Projectile`` so we know who shot what.

```js
game.addChild(new Projectile(
    this.x + this.width,
    this.y + this.height/2,
    this //this will let the projectile have a reference to the junkership that shot it!
))
```

```js
export default class Projectile extends Pixi.Sprite {
    constructor(x, y, junkership) {
        ...other code goes here...
        this.junkership = junkership
```

And that way, when we remove the ``Projectile`` on collision, we can just say ``projectile.junkership.score++``